mode/verilog/verilog.js

```diff
@@ -17,7 +17,8 @@ CodeMirror.defineMode("verilog", function(config, parserConfig) {
       statementIndentUnit = parserConfig.statementIndentUnit || indentUnit,
       dontAlignCalls = parserConfig.dontAlignCalls,
       noIndentKeywords = parserConfig.noIndentKeywords || [],
-      multiLineStrings = parserConfig.multiLineStrings;
+      multiLineStrings = parserConfig.multiLineStrings,
+      vxHooks = parserConfig.vxHooks || {}; // vx- verilog extensions
```

Comment:
I'd call this `hooks`, since it could also be used by other dialects, and there's nothing vx-specific about it.